src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.storage;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<Fruit, Integer> fruits = new HashMap<>();
+}
```

Comment:
maybe it will be better if u will use some StorageService class to have action with ur storage instead of directly access

Reply:
There was no requirements to use DAO.